wasm/javascript-v8/benchmark.md

```diff
@@ -0,0 +1,39 @@
+# Substreams Javascript runtime integration 
```

Comment:
(Comment on package name)

Let's refactor so package location is `wasm/v8` instead of `wasm/javascript-v8`